lib/lightning/config/bootstrap.ex

```diff
@@ -251,16 +254,79 @@ defmodule Lightning.Config.Bootstrap do
            env!("INIT_PROJECT_FOR_NEW_USER", &Utils.ensure_boolean/1, false)
 
     # To actually send emails you need to configure the mailer to use a real
-    # adapter. You may configure the swoosh api client of your choice. We
-    # automatically configure Mailgun if an API key has been provided. See
-    # https://hexdocs.pm/swoosh/Swoosh.html#module-installation for more details.
-    if api_key = env!("MAILGUN_API_KEY", :string, nil) do
-      config :lightning, Lightning.Mailer,
-        adapter: Swoosh.Adapters.Mailgun,
-        api_key: api_key,
-        domain: env!("MAILGUN_DOMAIN", :string)
+    # adapter. You may configure the swoosh api client of your choice.
+    # See # https://hexdocs.pm/swoosh/Swoosh.html#module-installation for more details.
+    if mail_provider = env!("MAIL_PROVIDER", :string, nil) do
+      case mail_provider do
```

Comment:
@stuartc Is it worth changing this to:

```
case env!("MAIL_PROVIDER", :string, nil)
```

And then handle `nil` as a no-op? It would allow the saving of one level of nesting.

Reply:
Ah good spotting, I was in the middle of making the `EMAIL_ADMIN` a required env variable in specific situations but backed out and left that behind. Will change it.